anura/avss/client.py

```diff
@@ -129,14 +163,17 @@ async def __aenter__(self):
     async def __aexit__(self, exc_type, exc, tb):
         pass
 
-    def _callback_and_generator(self) -> tuple[Callable[[Report], None], AsyncGenerator[Report, None]]:
+    def _callback_and_generator(self, parse) -> tuple[Callable[[Report], None], AsyncGenerator[Report, None]]:
         # Queue to hold the incoming reports
         reports: asyncio.Queue[Report] = asyncio.Queue()
```

Comment:
If I've understood things correctly, this type annotation is now violated when `XReport`s, e.g. `SnippetReport`s and `AggregatedValuesReport`s, are added to the queue since they are no longer subclasses of `Report`